lib/plugins/input/kubernetesEvents.js

```diff
@@ -1,45 +1,55 @@
 // Emit k8s events as Node.js events
+const { KubeConfig } = require('kubernetes-client')
 const Client = require('kubernetes-client').Client
-const config = require('kubernetes-client').config
-const JSONStream = require('json-stream')
-var consoleLogger = require('../../util/logger.js')
-var EventEmitter = require('events').EventEmitter
+const Request = require('kubernetes-client/backends/request')
+const consoleLogger = require('../../util/logger.js')
+const EventEmitter = require('events').EventEmitter
 
 class KubernetesEventEmitter extends EventEmitter {
   constructor () {
     super()
+    const kubeconfig = new KubeConfig()
     // do we run in k8s cluster?
     if (process.env.KUBERNETES_PORT_443_TCP !== undefined) {
```

Comment:
Is it better/cleaner to have just `process.env.KUBERNETES_PORT_443_TCP` instead of the whole `process.env.KUBERNETES_PORT_443_TCP !== undefined` ?

Reply:
I think the check for undefined is better, to avoid JavaScript 'fun' for values like 1, 0 or "".  
```
> process.env.FUN
undefined
> process.env.FUN=""
''
> false == process.env.FUN
true
> process.env.FUN=0
0
> false == process.env.FUN
true
> false === process.env.FUN
false
```

Reply:
I will leave it as it is. 

```
> process.env.KUBERNETES_TCP_PORT_443=""
''
> !!process.env.KUBERNETES_TCP_PORT_443
false
```
Above shows that the env var could be set to empty string, but still the existance of the env var is the indicator that LA runs in k8s.

Reply:
Wow, that is strange. Ok.